Makefile

```diff
@@ -18,17 +18,17 @@ run: ## run app
 	- make stop
 	- make start
 	- make install
-	- make build
+	- make build_dev
 
 start: ## start docker containers
-	- docker-compose up -d
+	- docker-compose up --build -d
 
 stop: ## stop docker containers
 	- docker-compose down
 
-dc_build: ## rebuild docker compose containers
+dc_build_prod: ## rebuild docker compose containers
 	- make .docker_img_deps
-	- docker-compose up --build
+	- docker-compose -f docker-compose.prod.yml up --build
```

Comment:
we don't have docker swarm on prod, having the prod version will not change the way we deploy to prod.